src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -289,6 +289,15 @@
             "Should only be used with SECOND_INPUT. ", optional = true)
     public File SECOND_INPUT_SAMPLE_MAP;
 
+    @Argument(doc = "A tsv with two columns representing the individual with which each sample is associated.  The first column is the sample id, and the second " +
+            "column is the associated individual id.  Values in the first column must be unique.  If INPUT_SAMPLE_MAP or SECOND_INPUT_SAMPLE_MAP is also specified, " +
+            "then the values in the first column of this file should be the sample aliases specified in the second columns of INPUT_SAMPLE_MAP and SECOND_INPUT_SAMPLE_MAP, " +
+            "respectively.  When this input is specified, expectations for matches will be based on the equality or inequality of the individual ids associated with two " +
+            "samples, as opposed to the sample ids themselves.  Samples which are not listed in this file will have their sample id used as their individual id, for the " +
+            "purposes of match expectations.  This means that one sample id could be used as the individual id for another sample, but not included in the map itself, and " +
+            "these two samples would be considered to have come from the same individual.", optional = true)
+    public File SAMPLE_INDIVIDUAL_MAP;
+
     @Argument(doc = "An argument that controls how crosschecking with both INPUT and SECOND_INPUT should occur. ")
```

Comment:
so if you are not going to modify the GROUP_BY=SAMPLE argument, could you add a note there (or in the enum) indicating that the value used for grouping will be the original, unmapped sample id?

Reply:
added a note in the documentation to clarify this.